### ISSUE TITLE
General login issue Internet Explorer 11

### ISSUE THREAD

**Issue by JoakimFalk** [PHPFusion/PHPFusion#947]
You simply can´t login with IE 11.

**Comment by meangczac** [comment]
My spider sense.. is cookie policy. If IE no cookie / windows settings off, how to login. They require a Compact Cookie Policy. http://stackoverflow.com/questions/4337876/create-a-compact-privacy-policy
Set security to lowest, and if cannot login, then it's our problem.

http://www.timeanddate.com/custom/cookiesie.html#ie9
It will cost money to put it in, and more technical knowledge of how to do it. 

A simpler way even: http://www.iubenda.com/en/features

But, I'll check it out for a hack around it.

This issue started from PHP-Fusion 6. I know  this first hand because this problem in IE introduced me to uninstall IE and use Netscape in the first place ;) However, IE is still good stuff. If you know how to handle one. 

Update, downloaded Internet Explorer 11 build 11.20 - a login functional on localhost. I can also login beta. Will test login panel

**Comment by meangczac** [comment]
![login_ok_ie11](https://cloud.githubusercontent.com/assets/4078041/17454029/891fe568-5bbb-11e6-8374-f9f1f785848f.JPG)

See I got no problem logging in using IE11. If it's not working, it's not PF code problem. I trashed all IE settings to High Risk. Issue with IE itself I think. 

Let's just address extras support with AOL browser, Opera or IE in support forum. Help them identify and troubleshoot. https://en.wikipedia.org/wiki/List_of_web_browsers
